spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -1510,9 +1519,11 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
                     batchSize: Int,
                     trigger: Trigger,
                     valRdd: JavaRDD[Sample],
-                    vMethods: JList[String]): Unit = {
+                    vMethods: JList[String],
+                    criterion: String = null,
```

Comment:
`criterion` and `embeddedCri` added here for the `loss` validation method only. We might need to convert `vMethods` from `List[String]` to `List[ValidationMethod]` instead of hacking here. You can take a look at the `SGD(JavaValue)` for how to warp those.